2021-04-24 06:38:40,664 INFO    MainThread:2588 [wandb_setup.py:_flush():69] setting env: {}
2021-04-24 06:38:40,666 INFO    MainThread:2588 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-24 06:38:40,667 INFO    MainThread:2588 [wandb_init.py:_log_setup():336] Logging user logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210424_063840-2ctyma01/logs/debug.log
2021-04-24 06:38:40,667 INFO    MainThread:2588 [wandb_init.py:_log_setup():337] Logging internal logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210424_063840-2ctyma01/logs/debug-internal.log
2021-04-24 06:38:40,667 INFO    MainThread:2588 [wandb_init.py:init():369] calling init triggers
2021-04-24 06:38:40,667 INFO    MainThread:2588 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-24 06:38:40,667 INFO    MainThread:2588 [wandb_init.py:init():418] starting backend
2021-04-24 06:38:40,668 INFO    MainThread:2588 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-24 06:38:40,720 INFO    MainThread:2588 [backend.py:ensure_launched():123] starting backend process...
2021-04-24 06:38:40,757 INFO    MainThread:2588 [backend.py:ensure_launched():127] started backend process with pid: 2658
2021-04-24 06:38:40,760 INFO    MainThread:2588 [wandb_init.py:init():423] backend started and connected
2021-04-24 06:38:40,763 INFO    MainThread:2588 [wandb_init.py:init():465] updated telemetry
2021-04-24 06:38:40,764 INFO    MainThread:2588 [wandb_init.py:init():484] communicating current version
2021-04-24 06:38:42,686 INFO    MainThread:2588 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-24 06:38:42,687 INFO    MainThread:2588 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-24 06:38:42,837 INFO    MainThread:2588 [wandb_init.py:init():522] starting run threads in backend
2021-04-24 06:38:46,151 INFO    MainThread:2588 [wandb_run.py:_console_start():1524] atexit reg
2021-04-24 06:38:46,154 INFO    MainThread:2588 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-24 06:38:46,155 INFO    MainThread:2588 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-24 06:38:46,157 INFO    MainThread:2588 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-24 06:38:46,157 INFO    MainThread:2588 [wandb_init.py:init():546] run started, returning control to user process
2021-04-24 06:38:46,160 INFO    MainThread:2588 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-24 06:38:46,163 INFO    MainThread:2588 [wandb_run.py:_restore():1466] restore
2021-04-24 06:38:48,306 INFO    MainThread:2588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 555
  total_bytes: 555
}

2021-04-24 06:38:48,896 INFO    MainThread:2588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 555
  total_bytes: 915
}

2021-04-24 06:38:49,038 INFO    MainThread:2588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 555
  total_bytes: 6043
}

2021-04-24 06:38:49,141 INFO    MainThread:2588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2021-04-24 06:38:49,243 INFO    MainThread:2588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2021-04-24 06:38:49,351 INFO    MainThread:2588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2021-04-24 06:38:49,454 INFO    MainThread:2588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2021-04-24 06:38:49,557 INFO    MainThread:2588 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6043
  total_bytes: 6043
}

2021-04-24 06:38:51,028 INFO    MainThread:2588 [wandb_run.py:_show_files():1838] logging synced files
